src/components/AvatarGroup/AvatarGroup.st.css

```diff
@@ -0,0 +1,129 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback;
+}
+
+:import {
+    -st-from: "wix-ui-core/index.st.css";
+    -st-named: Avatar;
+}
+
+:vars {
+    /*Defaults*/
+    DefaultAvatarBorderColor: color-1;
+}
+
+:vars {
+    /*Overrides*/
+     AvatarBorderColor: --overridable;
+}
+
+:vars {
+    avatarBorderColorValue: fallback(value(AvatarBorderColor), value(DefaultAvatarBorderColor));
+}
+
+.root {
+    -st-states: size(enum(large, medium, small, xSmall, xxSmall));
+    display: inline-flex;
+    flex-direction: row;
+}
+
+.root:size(large) {
+    padding-left: 10px;
+}
+
+.root:size(medium) {
+    padding-left: 8px;
+}
+
+.root:size(small) {
+    padding-left: 6px;
+}
+
+.root:size(xSmall) {
+    padding-left: 4px;
+}
+
+.root:size(xxSmall) {
+    padding-left: 2px;
+}
+
+.root .avatarContainer {
+    overflow: hidden;
+    position: relative;
+}
+
+.root:size(large) .avatarContainer {
+    margin-left: -10px;
+}
+
+.root:size(medium) .avatarContainer {
+    margin-left: -8px;
+}
+
+.root:size(small) .avatarContainer {
+    margin-left: -6px;
+}
+
+.root:size(xSmall) .avatarContainer {
+    margin-left: -4px;
+}
+
+.root:size(xxSmall) .avatarContainer {
+    margin-left: -2px;
+}
+
+.avatar {
+    -st-extends: Avatar;
+}
+
+.root .avatar::content {
+    border: solid 1px;
+    border-color: color(value(avatarBorderColorValue));
+}
+
+.root:size(small) .avatar::content,
+.root:size(medium) .avatar::content,
+.root:size(large) .avatar::content {
+    border-width: 2px;
+}
+
+.root:size(large) .avatar::content {
+    border-radius: 26px;
```

Comment:
I see in the storybook, that the border radius isn't working so good.
What I think we can do, is instead of defining all the different `border-radius`'s let's add:
```border-radius: 50%;``` to `.root::content`
It should work for all the avatar sizes

Reply:
Working on this :)